### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by Riceyo**
fixing on-prem location

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -46,10 +46,10 @@ def createMsg(data):
     elif fileExtn == "zip" and fileType == "dd":
         msg["description"] = 'Data Delivery files uploaded to GCP bucket from Blaise5'
         msg["dataset"] = 'blaise_dde'
-        msg["iterationL1"] = os.getenv('ON-PREM-SUBFOLDER')
-        msg["iterationL2"] = data['name'][3:7].upper()
-        msg["iterationL3"] = data['name'][7:8].upper()
-        msg["iterationL4"] = ''
+        msg["iterationL4"] = 'SYSTEMS'
```
Comment (jedimasterjonny) [inline main.py:49]: Should this be iterationL4? There's now no iterationL1, and L4 twice?

**Review by jedimasterjonny — CHANGES_REQUESTED** [review]
Think there's a mistake, I could be misunderstanding though.

File: main.py (hunk 1 above)
Comment (Riceyo) [reply main.py]: eek, well spotted

File: main.py [hunk 2]
```diff
@@ -46,10 +46,10 @@ def createMsg(data):
     elif fileExtn == "zip" and fileType == "dd":
         msg["description"] = 'Data Delivery files uploaded to GCP bucket from Blaise5'
         msg["dataset"] = 'blaise_dde'
-        msg["iterationL1"] = os.getenv('ON-PREM-SUBFOLDER')
-        msg["iterationL2"] = data['name'][3:7].upper()
-        msg["iterationL3"] = data['name'][7:8].upper()
-        msg["iterationL4"] = ''
+        msg["iterationL1"] = 'SYSTEMS'
+        msg["iterationL2"] = os.getenv('ON-PREM-SUBFOLDER')
+        msg["iterationL3"] = data['name'][3:6].upper()
```
Comment (jedimasterjonny) [inline main.py:51]: Was this meant to change from 3:7 to 3:6?

If the name of the file start OPN2004 for example, won't that make iterationL3 200, not 2004?

**Review by jedimasterjonny — CHANGES_REQUESTED** [review]

File: main.py (hunk 2 above)
Comment (jedimasterjonny) [reply main.py:51]: Just double checked, the Python : operator is exclusive of the last value, so that will only pull out 3 characters, not 4.

**Review by jedimasterjonny — APPROVED** [review]
Confirmed correct behaviour - LGTM

**Pull request merged**